sklearn/utils/estimator_checks.py

```diff
@@ -93,7 +93,7 @@ def _yield_non_meta_checks(name, estimator):
         # cross-decomposition's "transform" returns X and Y
         yield check_pipeline_consistency
 
-    if name not in ['SimpleImputer', 'Imputer']:
+    if name not in ['SimpleImputer', 'Imputer', 'MICEImputer']:
```

Comment:
Oh sorry I missed it. Could you change similarly to:

https://github.com/scikit-learn/scikit-learn/pull/10437/files#diff-a95fe0e40350c536a5e303e87ac979c4R75

and just add the MICEImputer there

Reply:
Done.